### PR TITLE
docs: v0.20 release

### PR DESCRIPTION
**Pull request by grutt**
# Description

Release Notes for v0.20

## Type of change

- [x] Documentation change (pure documentation change)

### CODE REVIEW

File: frontend/docs/pages/launches/rate-limits.mdx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+Now you can cancel a workflow run with a single click:
+
+![Untitled](/cancel-in-prog.gif)
+
+[Docs →](https://www.notion.so/Release-v0-20-e1c2cfb4569d41aaba9f4a4cd8919a7d?pvs=21)
+
+### Smaller Improvements
+
+- Github SSO option on [Hatchet Cloud](https://cloud.onhatchet.run/)
```
Comment (abelanger5) [inline frontend/docs/pages/launches/rate-limits.mdx:145]: This option exists for self-hosted too, just not well documented.

File: frontend/docs/pages/launches/rate-limits.mdx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+- [Interactive Tutorial](https://cloud.onhatchet.run/onboarding/get-started) to quickly learn the fundamentals of getting started with Hatchet
```
Comment (abelanger5) [inline frontend/docs/pages/launches/rate-limits.mdx:146]: People won't be redirected to this after login - and this exists on self-hosted too, right?

**Review by abelanger5 — APPROVED** [review]

File: frontend/docs/pages/launches/rate-limits.mdx (hunk 1 above)
Comment (grutt) [reply frontend/docs/pages/launches/rate-limits.mdx:145]: yeah... intentionally left out for now :)

File: frontend/docs/pages/launches/rate-limits.mdx (hunk 2 above)
Comment (grutt) [reply frontend/docs/pages/launches/rate-limits.mdx:146]: yeah i thought about this too, i think just linking is the best case. if they're signed in they will see the tutorial. if they're not signed in they'll need to sign up and then see the tutorial.

also on self-host but i think we focus on cloud here

**Pull request merged**